py/kubeflow/tf_operator/release.py

```diff
@@ -169,7 +169,7 @@ def build_operator_image(root_dir,
   # List of paths to copy relative to root.
   sources = [
     "build/images/tf_operator/Dockerfile", "examples/tf_sample/tf_smoke.py",
-    os.path.join(go_path, bin_path, "tf-operator.v1"),
+    os.path.join(go_path, bin_path, "training-operator.v1"),
```

Comment:
Do we need `release.py` file ? We probably need to check which files we are using from `py/kubeflow/tf_operator`.
cc @kubeflow/wg-training-leads

Reply:
The only release.py I can see referenced from the docs is https://github.com/kubeflow/tf-operator/blob/master/docs/release/release.py

Reply:
This file doesn't use `py/kubeflow/...`, right ?

Reply:
it does not.

Reply:
We are currently not using this script anymore. I think it should be ok to remove `tf-operator` reference first. Another story is needed to clean up  test/release tools